docker-ci.sh

```diff
@@ -193,6 +193,9 @@ function run_test_scss {
     npx gulp generate-scss
     npm run build-themes
     node build/gulp/scss/tests/identical.test.js
+
+    cd themebuilder-scss
+    npm i && npm run build && npm run test
```

Comment:
What do you think about moving the `lint` script call here?
https://github.com/DevExpress/DevExtreme/blob/8794fe2189f5c419b6f4a6840f6a2689e2ea07f6/docker-ci.sh#L10-L13

Reply:
in this case we need to run 'npm i' in themebuilder-scss folder in both tasks - linter and themebuilder tests.

Reply:
Got it.